HoaryFox/KarambaConnect/Element.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using Karamba.CrossSections;
+using Karamba.Elements;
+using Karamba.Geometry;
+using Karamba.Utilities;
+using STBReader;
+using STBReader.Member;
+using STBReader.Model;
+
+namespace HoaryFox.KarambaConnect
+{
+    public class Element
+    {
+        public static List<BuilderBeam> BuilderBeams(StbData stbData, List<string>[] k3Ids)
+        {
+            var logger = new MessageLogger();
+            var k3d = new KarambaCommon.Toolkit();
+            var outNodes = new List<Karamba.Geometry.Point3>();
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'outNodes'.](https://app.codacy.com/gh/hrntsm/HoaryFox/pullRequest?prid=6405207)

Reply:
remove outNodes